Angular/src/app/commentary/commentary.component.html

```diff
@@ -100,6 +100,13 @@ <h6>
     ">
   </ng-container>
   <!-- Show the fragment metrical analysis -->
+  <ng-container
+    *ngTemplateOutlet="
+      show_fragment_content;
+      context: { current_document_content: given_fragment.bibliography, title: 'Bibliography' }
+    ">
+  </ng-container>
+  <!-- Show the document bibliography -->
```

Comment:
Beetje off-topic mss, maar t is wmb wat verwarrend om de comment onder t bijpassende blok code te hebben.

Reply:
Jep. Neem ik mee in de volgende commentary rewrite :D